backend/core/views.py

```diff
@@ -293,3 +298,31 @@ def competence_reviews_outstanding(self, request, pk=None):
 #             result[team.id]["permissions"].append(permission)
 
 #     return Response(result)
+
+
+@staff_member_required
+def bulk_add_users_to_team(request, team_id):
```

Comment:
If you're up for it, you can refactor this to use a Class Based View instead, particularly `FormView` it could make our code cleaner and a lot less boilerplate-ish. Not a hard requirement though.